### PR TITLE
fix image_id alignment in coco datagen

### PR DESCRIPTION
**Pull request by changhiskhan**

### CODE REVIEW

File: python/benchmarks/coco/datagen.py [hunk 1]
```diff
@@ -33,21 +36,14 @@ def _get_instances_json(self, split):
...
-        annotations_df = df.merge(category_df, on="category_id")
+        """
+        Read instances and join to images
+        """
+        annotations_df = self._ann_to_df(instances_json)
         anno_df = (
             pd.DataFrame(
                 {
-                    "image_id": df.image_id,
+                    "image_id": annotations_df.image_id,
```
Comment (eddyxu) [inline python/benchmarks/coco/datagen.py:46]: is this the fix? while the rest are cleanups?

Comment (changhiskhan) [reply python/benchmarks/coco/datagen.py:46]: yup, one line fix 🤦

**Review by eddyxu — APPROVED** [review]
LGTM. Pending CI

**Comment by changhiskhan** [comment]
CI should be fixed after #293

**Pull request merged**